gremlin/README.md

```diff
@@ -15,9 +15,11 @@ their default values. See values.yaml for all available options.
 |       Parameter                        |           Description                                          |                         Default                                             |
 |----------------------------------------|----------------------------------------------------------------|-----------------------------------------------------------------------------|
 | `image.pullPolicy`                     | Container pull policy                                          | `Always`                                                                    |
+| `image.pullSecret`                    | Pull secrets for a private registry                            | `""` (When empty, no authentication is used)                                |
```

Comment:
@markshields-gl I'm calling this out for your visibility, I've renamed these pair of variables from `pullSecrets` to `pullSecret` (dropped the plural), since we're accepting just a single secret reference in this implementation.

Reply:
Thanks!  I was thinking the same but decided not to for the first iteration.